nano/templates/file_manager.tmpl

```diff
@@ -28,8 +28,8 @@
 					{{:helper.link('DELETE', null, { "PRG_deletefile" : value.name }, value.undeletable ? 'disabled' : null)}}
 					{{:helper.link('RENAME', null, { "PRG_rename" : value.name }, value.undeletable ? 'disabled' : null)}}
 					{{:helper.link('CLONE', null, { "PRG_clone" : value.name }, value.undeletable ? 'disabled' : null)}}
-					{{:helper.link('ENCRYPT', null, { "PRG_encrypt" : value.name }, value.undeletable ? 'disabled' : null)}}
-					{{:helper.link('DECRYPT', null, { "PRG_decrypt" : value.name }, value.undeletable ? 'disabled' : null)}}
+					{{:helper.link('ENCRYPT', null, { "PRG_encrypt" : value.name }, value.encrypted ? 'disabled' : null, value.undeletable ? 'disabled' : null)}}
```

Comment:
Here you could end up with two disabled tags.

Try something like that:
`
(value.encrypted || value.undeletable) ? 'disabled' : null
`

Reply:
shit, forgot